client/blocks/importer/wordpress/upgrade-plan/constants.tsx

```diff
@@ -49,23 +49,26 @@ export const UpgradePlanHostingTestimonials = [
 	},
 ];
 
-export const defaultHostingDetails = [
-	{
-		id: 'reduced-error-rate',
-		title: translate( 'Reduced error rate' ),
-		description: translate( '16% fewer errors' ),
-		icon: shield,
-	},
-	{
-		id: 'increased-speed',
-		title: translate( 'Increased speed' ),
-		description: translate( '30% faster' ),
+export const defaultHostingDetails: {
```

Comment:
Refactor it from array to object, as suggested in https://github.com/Automattic/wp-calypso/pull/90717#discussion_r1603326181.